src/App.tsx

```diff
@@ -17,18 +85,37 @@ export const App: React.FC = () => {
             <h1 className="title">Todos:</h1>
 
             <div className="block">
-              <TodoFilter />
+              <TodoFilter
+                handleChange={handleChange}
```

Comment:
```suggestion
                handleChange={setTodosStatus}
```
no need to create the `handleChange` wrapper above - it does nothing
the same for `queryChange`